modules/microshift-greenboot-check-status.adoc

```diff
@@ -34,3 +34,5 @@ $ systemctl show --property=ExecMainStatus --value greenboot-healthcheck.service
 ----
 $ cat /run/motd.d/boot-status
 ----
+
+//Q: is all of this still true?
```

Comment:
Yes, `microshift healthcheck` replaces bash functions that were previously used, but we still use greenboot as general framework.
It's just that instead of running bash function we run go code